container-host-files/etc/hcf/config/opinions.yml

```diff
@@ -148,6 +148,15 @@ properties:
       tag: cc
     db_scheme: mysql
     port: 3306
+  cf-usb:
+    broker:
+      username: broker-admin
+      port: 24054
+    management:
+      dev_mode: true
```

Comment:
When would we want not-dev-mode? (Presumably we don't want to ship it to customers with dev mode turned on?)